js/skin.js

```diff
@@ -123,19 +122,30 @@ var Skin = React.createClass({
    * @public
    * @description the function is called when we stop the rotation
    */
-  handleVrPlayerMouseUp: function() {
+  handleVrPlayerMouseUp: function(e) {
     if (this.props.controller && this.props.controller.isVrStereo) {
       return;
     }
     if (this.props.controller && this.props.controller.videoVr) {
+      var isVrMouseMove = this.state.isVrMouseMove;
+      if (Utils.isIos()) {
+        isVrMouseMove = false;
+      }
       this.setState({
         isVrMouseDown: false,
+        isVrMouseMove: isVrMouseMove,
         xVrMouseStart: 0,
         yVrMouseStart: 0
       });
+
       if (typeof this.props.controller.checkVrDirection === 'function') {
         this.props.controller.checkVrDirection();
       }
+
+      var endMove = this.state.isVrMouseMove || OO.isAndroid || OO.isIos;
```

Comment:
Can we add a comment about why this is true for Android and iOS?

Reply:
+

Reply:
add comment